data/json/proficiencies/melee_weapons.json

```
@@ -0,0 +1,778 @@
+    "time_to_learn": "8 h"
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_polearms_pro",
+    "category": "prof_combat",
+    "name": { "str": "Polearm Proficiency" },
+    "description": "You're proficient with the usage of most polearms.",
+    "can_learn": true,
+    "default_time_multiplier": 1.5,
+    "default_skill_penalty": 0.2,
+    "time_to_learn": "16 h",
+    "required_proficiencies": [ "prof_polearms_familiar" ]
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_polearms_master",
+    "category": "prof_combat",
+    "name": { "str": "Polearm Mastery" },
+    "description": "You've mastered the use of all kinds of polearms.",
+    "can_learn": true,
+    "default_time_multiplier": 1.5,
+    "default_skill_penalty": 0.2,
+    "time_to_learn": "32 h",
+    "required_proficiencies": [ "prof_polearms_pro" ]
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_bionics_familiar",
+    "category": "prof_combat",
+    "name": { "str": "Bionic Weapon Familiarity" },
+    "description": "You're familiar with how to use bionic weapons.",
+    "can_learn": true,
+    "default_time_multiplier": 1.5,
+    "default_skill_penalty": 0.2,
+    "time_to_learn": "8 h"
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_bionics_pro",
+    "category": "prof_combat",
+    "name": { "str": "Bionic Weapon Proficiency" },
+    "description": "You're proficient with the usage of most bionic weapons.",
+    "can_learn": true,
+    "default_time_multiplier": 1.5,
+    "default_skill_penalty": 0.2,
+    "time_to_learn": "16 h",
+    "required_proficiencies": [ "prof_claws_familiar" ]
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_bionics_master",
+    "category": "prof_combat",
+    "name": { "str": "Bionic Weapon Mastery" },
+    "description": "You've mastered the use of all kinds of bionic weapons.",
+    "can_learn": true,
+    "default_time_multiplier": 1.5,
+    "default_skill_penalty": 0.2,
+    "time_to_learn": "32 h",
+    "required_proficiencies": [ "prof_claws_pro" ]
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_spears_familiar",
+    "category": "prof_combat",
+    "name": { "str": "Spear Familiarity" },
+    "description": "You're familiar with how to hold most spears.",
+    "can_learn": true,
+    "default_time_multiplier": 1.5,
+    "default_skill_penalty": 0.2,
+    "time_to_learn": "8 h"
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_spears_pro",
+    "category": "prof_combat",
+    "name": { "str": "Spear Proficiency" },
+    "description": "You're proficient with the usage of most spears.",
+    "can_learn": true,
+    "default_time_multiplier": 1.5,
+    "default_skill_penalty": 0.2,
+    "time_to_learn": "16 h",
+    "required_proficiencies": [ "prof_spear_familiar" ]
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_spears_master",
+    "category": "prof_combat",
+    "name": { "str": "Spear Mastery" },
+    "description": "You've mastered the use of all kinds of constructed spears.",
```

```suggestion
    "description": "You've mastered the use of all kinds of spears.",
```
All-natural spears get out! 😛